.github/workflows/labeler.yml

```diff
@@ -10,4 +10,5 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Do we need this? Labeler already seems to be working, doesn’t it?

Reply:
Not really, I realized that you have to use this to make it actually work.